index.js

```diff
@@ -38,18 +56,28 @@
 
 // ***** Question 4 *****
 
-// *** Uncomment the lines below to test
+// function sameSameButDifferent(num, maybeNum) {
+//   if (num === maybeNum) {
+//     console.log("same same");
+//     } else if (num == maybeNum) { 
+//     console.log("same same (but different)");
+//   } else {
+//     console.log("different");
+//   }
```

Comment:
The logic for your conditionals looks good, but just like above, this function is meant to **return** something instead of `console.log`. Also, you can use `parseInt` so you an use the `===` instead of `==`.

```js
function sameSameButDifferent(num, maybeNum) {
  if (num === maybeNum) {
    return 'same same'
  } else if (num === parseInt(maybeNum)) {
    return 'same same (but different)'
  } else {
    return 'different'
  }
}
```